app/views/dependencies/_dependencies.html.erb

```diff
@@ -0,0 +1,27 @@
+<% if @dependencies.key?(scope) %>
+  <% if scope == "development" %>
+    <div id="<%=@latest_version.rubygem.name+scope%>"><span class="scope"><%= scope.titlecase %> :</span>
+    <div class="deps_scope deps_toggle">
+  <% else %>
+    <div id="<%=@latest_version.rubygem.name+scope%>"><span class="scope scope--expanded"><%= scope.titlecase %> :</span>
```

Comment:
Remove `if else` on scope and change span to following:
`<span class="scope <%= scope--expaned if scope == "development" %>">`.
Please also make sure there is space around there erb tag (<%= <> %>) everywhere.